addons/mod.py

```diff
@@ -98,7 +98,7 @@
     """,
 )
 
-hidden_term_line = ' • When you have finished reading all of the rules, send a message in this channel that includes the sha1 of your discord `name#discriminator`, and we\'ll grant you access to the other channels. You can find your `name#discriminator` under the discord channel list.'
```

Comment:
You shouldn't have removed the `\`, in this context it's an escape character that doesn't show up in the output but makes the `'` after it be part of the text instead of ending the text.

Reply:
Thank you!